demos/background_subtraction_demo/cpp_gapi/main.cpp

```diff
@@ -0,0 +1,215 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <monitors/presenter.h>
+#include <utils/args_helper.hpp>
+#include <utils_gapi/stream_source.hpp>
+#include <utils/config_factory.h>
+
+#include <opencv2/gapi/streaming/cap.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/core.hpp>
+
+#include <opencv2/gapi/fluid/core.hpp>
+#include <opencv2/gapi/fluid/imgproc.hpp>
+#include <opencv2/gapi/cpu/core.hpp>
+#include <opencv2/gapi/cpu/imgproc.hpp>
+
+#include <opencv2/highgui.hpp>
+
```

Comment:
```suggestion

#include <openvino/openvino.hpp>
```
Since, we use  `ov::get_openvino_version()`, add  `#include <openvino/openvino.hpp>`  explicitly